lambdas/src/apis/status/health.ts

```diff
@@ -29,7 +29,9 @@ export async function refreshContentServerStatus(
 
     const obtainDeploymentTimeIsTooLong = obtainDeploymentTime > ms(maxDeploymentObtentionTime)
 
-    if (hasOldInformation || obtainDeploymentTimeIsTooLong) {
+    const isBootstrapping = (serverStatus as any).synchronizationStatus?.synchronizationState === 'Bootstrapping'
```

Comment:
Hmm, I'd revert the condition, and consider the content server healthy if and only if `synchronizationState == "Syncing"` instead of `synchronizationState != "Bootstrapping"`. WDYT?

Reply:
Agree on that, anyway I don't love the semantic on that, as being `Bootstrapping` is a healthy status but not ready to receive traffic yet 🤔 .

Reply:
We also have the `"Synced"` [state](https://github.com/decentraland/catalyst/blob/97054c9f5ee093f61b978b9329d178039d930940/content/src/service/synchronization/SynchronizationManager.ts#L8-L12). Should I also include it? 

`synchronizationState == "Syncing" || synchronizationState == "Synced"`

Reply:
`Synced` was removed with the new sync. we may delete it from the enum. Returning "Bootstrapping" instead of "Unhealthy" also sounds reasonable in my head I'm ok with both statuses as long as readyToUse==false if it is Bootstrapping